src/App.js

```diff
@@ -16,17 +18,33 @@ const headerData = [
   },
 ];
 
-function App() {
+const App = () => {
+  const [isGoLoaded, setIsGoLoaded] = useState(false);
+
+  useEffect(() => {
+    WebAssembly.instantiateStreaming(
```

Comment:
Would this function ever fail? Might want to handle it gracefully. But since it's on the go code, then it probably doesn't matter too much.

Reply:
trueee, I'll add some error handling with some kinda error page, since if this part fails, none of the go stuff will work either